webapp/channels/src/reducers/views/scheduled_posts.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import type {AnyAction} from 'redux';
+import {combineReducers} from 'redux';
+
+import type {ScheduledPostsState} from '@mattermost/types/schedule_post';
+
+import {ActionTypes} from 'utils/constants';
+
+function byTeamId(state: ScheduledPostsState['byTeamId'] = {}, action: AnyAction) {
```

Comment:
Done.

Reply:
> Do you think we're ever going to need to access the scheduled posts individually by their IDs

Yes, I think we will, but I'm not 100% sure about if and how I'll need it which is why I left it as it for now.

Reply:
That's fine as long as plugins don't try to access this data directly since then we'll be breaking them. I think it's fine to assume they won't for this though